### ISSUE TITLE
False negative comprehension structure for `WPS361`

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#2075]
### What's wrong

This code does not raise a `WPS361` violation:

```python
rules = [
    rule for rule in
    config.get_option(self._rules_opt_name).items()
]
```

### How it should be

It should

### Flake8 version and plugins

`wps@0.15.3`

**Comment by sobolevn** [comment]
Ok, it is better to release this under `0.16.0`, because it may produce new violations.
